GeneratorInterface/PhotosInterface/test/DYnorad_Tune4C_13TeV_pythia8.py

```diff
@@ -0,0 +1,117 @@
+process.load('Configuration.StandardSequences.MagneticField_38T_cff')
+process.load('Configuration.StandardSequences.Generator_cff')
+process.load('IOMC.EventVertexGenerators.VtxSmearedRealistic8TeVCollision_cfi')
+process.load('GeneratorInterface.Core.genFilterSummary_cff')
+process.load('Configuration.StandardSequences.Validation_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1000)
+)
+
+# Input source
+process.source = cms.Source("EmptySource")
+
+process.options = cms.untracked.PSet(
+
+)
+
+# Production Info
+process.configurationMetadata = cms.untracked.PSet(
+    version = cms.untracked.string('$Revision: 1.19 $'),
+    annotation = cms.untracked.string('Configuration/Generator/python/ThirteenTeV/WToMuNu_M_500_Tune4C_13TeV_pythia8_cfi.py nevts:100000'),
+    name = cms.untracked.string('Applications')
+)
+
+# Output definition
+
+process.RAWSIMoutput = cms.OutputModule("PoolOutputModule",
+    splitLevel = cms.untracked.int32(0),
+    eventAutoFlushCompressedSize = cms.untracked.int32(5242880),
+    outputCommands = process.RAWSIMEventContent.outputCommands,
+    fileName = cms.untracked.string('DYnorad_Tune4C_13TeV_pythia8_cfi_py_GEN_VALIDATION.root'),
+    dataset = cms.untracked.PSet(
+        filterName = cms.untracked.string(''),
+        dataTier = cms.untracked.string('GEN')
+    ),
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('generation_step')
+    )
+)
+
+# Additional output definition
+
+# Other statements
+process.genstepfilter.triggerConditions=cms.vstring("generation_step")
+process.mix.playback = True
+process.mix.digitizers = cms.PSet()
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:startup', '')
+
+process.generator = cms.EDFilter("Pythia8GeneratorFilter",
+    ExternalDecays = cms.PSet(
+        Tauolapp114 = cms.untracked.PSet(
+            UseTauolaPolarization = cms.bool(True),
+            InputCards = cms.PSet(
+                mdtau = cms.int32(0),
+                pjak2 = cms.int32(3),
+                pjak1 = cms.int32(3)
+            )
+        ),
+        parameterSets = cms.vstring('Tauolapp114')
+    ),
+    UseExternalGenerators = cms.untracked.bool(True),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.0),
+    crossSection = cms.untracked.double(0.2188),
+    maxEventsToPrint = cms.untracked.int32(1),
+    PythiaParameters = cms.PSet(
+        processParameters = cms.vstring('Main:timesAllowErrors = 10000', 
+					'ParticleDecays:limitTau0 = on', 
+					'ParticleDecays:tauMax = 10', 
+					'Tune:ee 3', 
+					'Tune:pp 5',
+					'PartonLevel:FSR = off',
```

Comment:
I don't  recall this being the only flag to set off for norad, can I ask what is the specific purpose of this cfg?

Reply:
@perrozzi the idea is have two cfgs, one with photos++ enabled and one without it in order to compare the output and see the extra photons in there due to the photos++.

Reply:
ok, thanks (I had not noticed the same flag in the other cfg)